src/dist/replication/lib/mutation.h

```diff
@@ -59,6 +59,9 @@ class mutation : public ref_counter
     mutation();
     virtual ~mutation();
 
+    // copy mutation and create a new client_request which will not reply
```

Comment:
这注释不行啊，文不对意啊，`a new client_request` 这个就错了

```
Copy from an existing and committed mutation. The new mutation is allowed to be re-committed 
on replica (typically on child during partition-split), but is able to not reply 
to client (since this mutation is already replied).
```

Reply:
已update